pharos-cluster.gemspec

```diff
@@ -26,8 +26,8 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "kubeclient"
   spec.add_runtime_dependency "net-ssh", "4.2.0"
   spec.add_runtime_dependency "net-scp", "1.2.1"
-  spec.add_runtime_dependency "rbnacl", "4.0.2"
   spec.add_runtime_dependency "rbnacl-libsodium", "1.0.16"
+  spec.add_runtime_dependency "rbnacl", "4.0.2"
```

Comment:
Is there some reason we can't use 5.0.0 (June 13, 2017)?

Reply:
(otherwise it should be possible to just drop the rbnacl from gemspec, keeping just the `rbnacl-libsodium` brings 5.0.0)

Reply:
I did dig this from net-ssh issues... it might work with newer versions. Maybe we should check metasploit (or some other project that has net-ssh)?

Reply:
Net::SSH 5.0 beta drops `rbnacl` altogether and switches to `ed25519`.

Reply:
Looks like 4.2 requires `< 5.0  >= 3.2`

Reply:
(weird that bundler still agrees to install rbnacl 5.0.0 and net-ssh 4.2 if you change it in the gemspec)